vignettes/.gitignore

```
@@ -0,0 +1,4 @@
+*.html
+*.R
```

[nitpick] Ignoring all `*.R` under `vignettes/` may hide intentionally committed scripts; consider targeting only generated files (e.g., `*_files/*.R`).
```suggestion
*_files/*.R
```